queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -67,7 +67,21 @@ def _get_pdfs(
             attach_order += 1
 
         # add receipt
-        corp_name = business.get('legalName')
+        corp_name = filing.filing_json['filing']['continuationIn']['nameRequest'].get('legalName', 'Numbered Company')
```

Comment:
This is not what i was suggesting. Can you try this?
```
if not (corp_name := business.get('legalName')):
    legal_type = business.get('legalType')
    if legal_type:
        corp_name = Business.BUSINESSES.get(legal_type, {}).get('numberedDescription')
```

Reply:
If it works can you update in the below files as well?
https://github.com/bcgov/lear/blob/main/queue_services/entity-emailer/src/entity_emailer/email_processors/amalgamation_notification.py#L73
https://github.com/bcgov/lear/blob/main/queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py#L84

Reply:
thanks for pointing that out, yes, working on it now

Reply:
how about this (in case `business` is None):
```python
if not (corp_name := (business or {}).get('legalName')):
    legal_type = business.get('legalType')
    corp_name = Business.BUSINESSES.get(legal_type, {}).get('numberedDescription')
```

Reply:
business will never be None

Reply:
Good to know. gonna remove that `or {}`. Other than that, it works in my local Emailer run.

Reply:
updated

Reply:
Tested in local emailer successfully getting receipts using the updated code for:
- continuation in 
- amalgamation application
- incorporation application